.buildkite/pipeline.yml

```diff
@@ -0,0 +1,12 @@
+steps:
+  - label: ":hammer: Specs"
```

Comment:
the agents dedicated for this expose two flags: `m1=true, rbenv=true`
although its very likely there won't be other types of machines, you could add:
```yaml
agents:
      - "m1=true"
```

Reply:
@dcvz added tag.